docs/quick-start-guide.md

```diff
@@ -461,3 +464,82 @@ select * from carbon_table;
 
 **Note :** Create Tables and data loads should be done before executing queries as we can not create carbon table from this interface.
 
+## Installing and Configuring CarbonData on Hive
+### Build (In 1.2.0, hive integration only support spark2.1 and hadoop2.7.2)
```

Comment:
I think we should add one more in here, because it will difficult to maintain for it.
 I suggest you only add a link for it in here, it's enough.